build/teamcity/cockroach/ci/tests/unit_tests_impl.sh

```diff
@@ -3,6 +3,13 @@
 set -xeuo pipefail
 
 bazel build //pkg/cmd/bazci --config=ci
-$(bazel info bazel-bin --config=ci)/pkg/cmd/bazci/bazci_/bazci --config=cinolint --config=simplestamp --compilation_mode=fastbuild \
+
+if tc_release_branch; then
+  # enable up to 2 attempts per test executable to report flakes but only on release branches (i.e., not staging)
+  $(bazel info bazel-bin --config=ci)/pkg/cmd/bazci/bazci_/bazci --config=cinolint --config=simplestamp --compilation_mode=fastbuild \
 		                   test //pkg:small_tests //pkg:medium_tests //pkg:large_tests //pkg:enormous_tests -- \
-                                   --profile=/artifacts/profile.gz
+                                   --profile=/artifacts/profile.gz --flaky_test_attempts=3
+else 
+  $(bazel info bazel-bin --config=ci)/pkg/cmd/bazci/bazci_/bazci --config=cinolint --config=simplestamp --compilation_mode=fastbuild \
+                                   test //pkg:small_tests //pkg:medium_tests //pkg:large_tests //pkg:enormous_tests -- \
```

Comment:
Isn't something off here? There's a backslash at the end of this line, but nothing following it. Shouldn't we still be passing the `--profile` flag in this branch?

Related: instead of copying this long command, we could define the flaky argument as a shell variable and use it when invoking `bazci` here.

Reply:
> Isn't something off here? There's a backslash at the end of this line, but nothing following it. Shouldn't we still be passing the `--profile` flag in this branch?
> 

Yep, copy/paste error. I wish there were an easy way to test shell script changes.

> Related: instead of copying this long command, we could define the flaky argument as a shell variable and use it when invoking `bazci` here.

I thought about folding this into `bazci`. The downside is wrappers (e.g., `dev` and `bazci`) make it more difficult to comprehend what is actually being passed into `bazel`.

Reply:
I was actually suggesting something at the bash-level to avoid the copying and pasting of that long command, like:

```sh
if tc_release_branch; then
    flaky_attempts_arg="--flaky_test_attempts=3"
fi

./bazci etc etc $flaky_attempts_arg
```